pkg/asset/installconfig/alibabacloud/client.go

```diff
@@ -234,18 +234,46 @@ func (client *Client) ListDNSDomain(baseDomain string) (response *alidns.Describ
 }
 
 // ListPrivateZones gets the list of privatzones.
-func (client *Client) ListPrivateZones(zoneName string) (response *pvtz.DescribeZonesResponse, err error) {
+func (client *Client) ListPrivateZones(zoneName string, queryVpcId string) (response *pvtz.DescribeZonesResponse, err error) {
```

Comment:
I would rather these be two separate functions.
```
func (client *Client) ListPrivateZonesByName(zoneName string) (response *pvtz.DescribeZonesResponse, err error) {
```
```
func (client *Client) ListPrivateZonesByVPC(vpcId string) (response *pvtz.DescribeZonesResponse, err error) {
```

Reply:
Can't agree more. But I added two functions on the basis of ListPrivateZones